beacon_node/http_api/src/lib.rs

```diff
@@ -2138,6 +2138,11 @@ pub fn serve<T: BeaconChainTypes>(
                             query.slot,
                             &query.attestation_data_root,
                         )
+                        .map_err(|_e| {
+                            warp_utils::reject::custom_bad_request(
+                                "unable to fetch aggregate".to_string(),
+                            )
+                        })?
```

Comment:
This error's swallowed 

```suggestion
                        .map_err(|e| {
                            warp_utils::reject::custom_bad_request(format!(
                                "unable to fetch aggregate: {e:?}"
                            ))
                        })?
```

Reply:
Nice catch, thank you!